modules/500-openvpn/images/ovpn-admin/werf.inc.yaml

```diff
@@ -46,7 +46,7 @@ git:
 shell:
   install:
   - cd /src
-  - git clone {{ .SOURCE_REPO }}/flant/ovpn-admin.git .
+  - git clone {{ .SOURCE_REPO }}/palark/ovpn-admin.git .
   - git checkout {{ $gitCommit }}
   - git apply /patches/*.patch --verbose
   - echo {{ $gitCommit }} > version
```

Comment:
you need to add `rm -r .git` at the end

Reply:
I don't see anything similar in the neighboring modules. Why are we doing this, and why are we doing it only in this module? @alexey-gavrilov-flant

Reply:
I only suggested here because the rebuild will only be in this image. We will gradually add it to other images as well.